### PR TITLE
Start of adding Docker

### PR DESCRIPTION
**Pull request by exussum12**
Still WIP.

The API DB has a dependency on having the mysql command. Looking to work around that.
Just pushing what I have so far for comments

### CODE REVIEW

**Comment by exussum12** [comment]
Web2 and api work, Looking at legacy if someone else could test that would be great!

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+version: '3'
+services:
+  nginx-proxy:
+    image: jwilder/nginx-proxy
+    ports:
+      - "80:80"
```
Comment (IonBazan) [inline docker-compose.yml]: Consider using `.env` file to allow users customize the port number. This also applies to `db`.

**Comment by ericpoe** [comment]
@exussum12 I added the WIP tag to this just to remind reviewers that this is still a work-in-progress. I love where you're going with this. Docker all the things!

**Comment by svpernova09** [comment]
Thanks for your work on this, we're going to close this PR in favor of continuing docker work via https://github.com/joindin/joindin-docker

**Pull request closed without merging**